components/installer/pkg/config/v1/config.go

```diff
@@ -55,6 +55,7 @@ func (v version) Defaults(in interface{}) error {
 	cfg.Workspace.Runtime.FSShiftMethod = FSShiftShiftFS
 	cfg.Workspace.Runtime.ContainerDSocket = "/run/containerd/containerd.sock"
 	cfg.Workspace.Runtime.ContainerDRuntimeDir = "/var/lib/containerd/io.containerd.runtime.v2.task/k8s.io"
+	cfg.OpenVSX.URL = "https://open-vsx.org"
```

Comment:
Considering we're making this an explicit default, we should add config validation which ensures the value is an actual URL (e.g. using `url.Parse`)